chef_master/source/perform_compliance_scan.rst

```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
 
 #. Once you've made your selection, click **Get** to add the profiles to your profile collection. You will see them show up under **Profiles**.
 
+#. (Optional) This description will use two profiles for demonstration.  To follow along, get the following profiles:
+    - DevSec Linux Security Baseline
+    - DevSec Windows Security Baseline
+
 #. (Optional) If you want to upload a profile from your local machine, click **Upload**.
 
    Select your profiles to upload and click **Done** when you are finished.
 
    .. note:: Your profiles must be archived as tarballs with a ``.tar.gz`` extension.
 
-Use the audit cookbook on nodes in your cluster
--------------------------------------------------------
+Using the Audit Cookbook
+------------------------------------------------------
 
-Once you have the audit cookbook installed on your Chef server and your profiles are ready to use in Chef Automate, you must add the ``audit::default`` recipe to your nodes' run-lists. The audit cookbook can be reused for all nodes because all node-specific configuration is done through Chef attributes.
+The recommended method for using the audit cookbook is via a wrapper cookbook. A wrapper cookbook is a regular cookbook that includes recipes from other cookbooks. In case the wrapper cookbook will use ``include audit::default``.
```

Comment:
`In this case`? Also I would change `include audit::default` to just `audit::default` or `the default recipe of the audit cookbook`